ckanext/qdes_schema/helpers.py

```diff
@@ -644,6 +644,12 @@ def get_publish_activities(pkg):
                 # If failed, get the last success published date.
                 processed_date = get_last_success_publish_date(resource)
 
+            # Get detail.
+            try:
+                details = json.loads(resource_publish_log.details)
+            except Exception as e:
+                details = {}
```

Comment:
@awset We should probably log the exception here as a warning with the details of the resource_publish_log which will help investigate any potential issues in the future.

Reply:
Implemented warning log